### ISSUE TITLE
Project Proposal Feedback 

### ISSUE THREAD

**Issue by poojapathak1725** [COGS118A/Group043-Wi23#1]
~Project Proposal Grade: 6.75/9~

Updated Project Proposal Grade - 6.75 + 0.25 (Abstract) + 0.25 (Background) + 0.25 (Statement) + 0.5 (Data) + 1 (Solution) = 9/9

Title and Abstract (0.75/1) - 
 Please include an informative title for your project. Minor point - Please mention the model you plan to use to predict gross income. (-0.25)

Background (0.75/1)- 
Why did the prior work only result in an accuracy of 53%? Did they mention their limitations? Mention them. 
Did you draw inspiration from their work for any models/feature engineering techniques? (-0.25)

Fix formatting on citations - In your footnotes you have 3 sources, but you refer to 2 in your background. 

Research Problem Statement (& Significance, Purpose of Project)(1.25/1.5) - 
OK - I would also add the significance and purpose of the project in this section, and connect it to background literature. How will your project benefit anyone? Explain. (-0.25)

Guiding rubric - Identifies a relevant research issue. Research questions are succinctly stated, connected to the research issue, and supported by the literature. Variables and controls have been identified and described. Connections are established with the literature.

Specify which kind of loss you're trying to minimize. 

Data (1/1.5)- 
OK. Are there any other feature engineering techniques you plan to use? Mention other pre-processing steps - dropping null values, checking outliers, checking for multicollinearity between features etc. How will you handle these? (-0.5)

Proposed Solution (0.5/1.5)

Since this seems like a relatively straightforward problem, I will likely need to see extensive EDA, feature selection techniques etc. 

As I mentioned earlier - Check that the linear assumption holds using scatterplots etc. Check for multicollinearity. Check for outliers and handle them. 
For the scope of this class, you NEED to do more than linear regression. You should try XGBoost or other regression models and compare them and use cross-validation.  Additionally, specify what kind of cross-validation and why you think it's appropriate given the size and nature of your dataset.
Are you using grid search to tune hyperparameters? Mention details (-1)

Metrics (1.5/1.5)
 MSE and RMSE are good metrics - but what other kinds of metrics can you use for a regression analysis? Explore them as well. 

Ethics and Privacy (0.5/0.5) - 
Good - Mention how you will deal with confounding variables in your analysis. 

Team Expectations (0.25/0.25)- 
OK. 

Timeline (0.25/0.25) - 
OK. 

Other Comments - Good topic, but make sure you compare and contrast a couple of different kinds of regression models for this problem. 

You can reply to this feedback below. Make sure to incorporate the feedback mentioned here so you can make up these points in the Checkpoint. Contact me anytime if you want help improving your project or have any questions at all!
